sympy/solvers/deutils.py

```diff
@@ -206,8 +224,18 @@ def _desolve(eq, func=None, hint="default", ics=None, simplify=True, *, prep=Tru
     # being called more than it needs to be by passing its results through
     # recursive calls.
     if kwargs.get('classify', True):
-        hints = classifier(eq, func, dict=True, ics=ics, xi=xi, eta=eta,
-        n=terms, x0=x0, hint=hint, prep=prep)
+        hints = classifier(
+            eq,
+            func,
+            dict=True,
+            ics=ics,
+            xi=xi,
+            eta=eta,
+            n=terms,
+            x0=x0,
+            hint=hint,
+            prep=prep
+        )
```

Comment:
I don't really like this change. Maybe I would make a `**opts` dict or something but I'm not sure.

Reply:
The line breaks are there because of the 79 column limit. If you set it to something like 100, it becomes:
```python
        hints = classifier(
            eq, func, dict=True, ics=ics, xi=xi, eta=eta, n=terms, x0=x0, hint=hint, prep=prep
        )
```

(I would stick with PEP8's 79, but it illustrates that arguments are put on one line if possible.)

The dedenting is the same effect as you noted here https://github.com/sympy/sympy/pull/22437#discussion_r744184272. But yapf's default profile ("`pep8`") looks worse imo:
```python
        hints = classifier(eq,
                           func,
                           dict=True,
                           ics=ics,
                           xi=xi,
                           eta=eta,
                           n=terms,
                           x0=x0,
                           hint=hint,
                           prep=prep)
```

Note that the same happens in function with many arguments:
https://github.com/sympy/sympy/blob/357fae944d1804df60ab5747a00b2d068d0464a7/sympy/solvers/deutils.py#L143-L152

For function definitions, this style is quite helpful when type hints are added.

Reply:
Type hints may change it, but I'm not a fan of this either. OK if it re-breaks the lines to have 80 characters, but having a separate line for each argument... (similar to the init-files)